src/ucp/wireup/wireup_cm.c

```diff
@@ -915,11 +1118,14 @@ static void ucp_cm_server_conn_notify_cb(uct_ep_h ep, void *arg,
 
 ucs_status_t ucp_ep_cm_connect_server_lane(ucp_ep_h ep,
                                            uct_listener_h uct_listener,
-                                           uct_conn_request_h uct_conn_req)
+                                           uct_conn_request_h uct_conn_req,
+                                           ucp_rsc_index_t cms_used_idx)
```

Comment:
better to pass cm object directly since cm index is related to worker which is absent in arg list

Reply:
you mean to pass `uct_cm_h cm`?
why not pass the index? we get the worker from the ep.

Reply:
yes, because there is no worker in args
> we get the worker from the ep.

it makes API of this function not clear, what if we remove worker from EP sometime

Reply:
the cm_idx that is passed here is taken from `ucp_conn_request_h conn_request` (same as `uct_conn_req` and `listener` before it).
if we remove worker from ep sometime, Many other changes would need to be made in ucp.
if a worker is available from the ep now, what not use it?